spam-villa/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+KEYS=$(paste -sd, testnet-keys.txt)
+
+
+# Optional: -e ETHEREUM_THROW_CONTRACT=0xffc4700dc5ac0639525ab50ab0a84ac125599f33
+# Uses hardcoded values per network-id if missing
+# KEYS is a comma-separated list of 6 private keys
+
+docker run -t --net host -e ETHEREUM_PRIVATE_KEYS=$KEYS -e ETHEREUM_NODE_HOST=127.0.0.1 augur/spam-villa
```

Comment:
I would prefer a `docker-compose.yml` over a shell script for this.  That way it will work on any OS.

Also, forcing to `--net host` seems unnecessary in this case (we don't need a ton of bandwidth or the extra microseconds of latency), and it introduces complexity to running that the default docker networking doesn't (like port conflicts with host system).

Lastly, why `-t`?  I have never used it without an accompanying `-i`, I'm not actually sure what it does.  :)

Reply:
I didn't use a compose because of the way i wanted to the testnet key file external, do you have a good solution for this?

`--net host` accidentally left in.

`-t` allows you to Ctrl-C it

Reply:
`env_file` i'll give that a shot

Reply:
I think this actually speaks to a deeper issue in that we (Augur) don't have a real key management solution.  I suspect that we won't be able to solve that here in this PR, but an env file may be a reasonable workaround to get it working in docker-compose.  Alternatively (and a bit hacky) just 6 environment variables (since the array isn't actually dynamic, having them in a single environment variable doesn't really add much value IMO).